scripts/docker_files/docker_file_ci_centos_7/DockerFile

```diff
@@ -4,28 +4,43 @@ USER root
 
 ENV HOME /root
 
-#RUN python --version
+# Enable devtools packages
+RUN yum -y install centos-release-scl
+RUN yum-config-manager --enable rhel-7-server-devtools-rpms
+RUN yum-config-manager --enable rhel-7-server-extras-rpms
+RUN yum-config-manager --enable rhel-7-server-optional-rpms
+RUN yum-config-manager --enable rhel-server-rhscl-7-rpms
+
+RUN yum -y update
+
+# Install dependencies
 RUN yum -y --quiet install wget git cmake gcc-c++ unzip python-pip make gcc-gfortran zlib-devel bzip2-devel openssl-devel samba-client openmpi-devel hdf5-openmpi-devel boost-devel
 
+# Install scl-utils
+RUN yum -y install scl-utils
+
+# Devtools 8
+RUN yum -y install devtoolset-8 devtoolset-8-gcc-c++
+RUN yum -y install binutils-devel wget && rm -rf /var/cache/yum/*
+RUN source scl_source enable devtoolset-8
 
-# compiling python3.5.4 to be compatible with HW
+# compiling python3.8.10 to be compatible with HW
 RUN cd /usr/src
-RUN wget https://www.python.org/ftp/python/3.5.4/Python-3.5.4.tgz
-RUN tar xzf Python-3.5.4.tgz -C /usr/src
-RUN ls /usr/src/Python-3.5.4
-RUN sh /usr/src/Python-3.5.4/configure --enable-optimizations --with-lto --enable-shared LDFLAGS="-Wl,-rpath /usr/local/lib"
+RUN wget https://www.python.org/ftp/python/3.8.10/Python-3.8.10.tgz
+RUN tar xzf Python-3.8.10.tgz -C /usr/src
+RUN ls /usr/src/Python-3.8.10
+RUN sh /usr/src/Python-3.8.10/configure --enable-optimizations --with-lto --enable-shared LDFLAGS="-Wl,-rpath /usr/local/lib"
 RUN make altinstall
-RUN rm Python-3.5.4.tgz
-RUN python3.5 -m pip install 'cx_Freeze==6.1' numpy sympy
```

Comment:
you are not installing numpy and sympy? I think we use them in some places in the code

Reply:
Oh good catch, I forgot to re-add them for 3.8